docker-compose-dev.yml

```diff
@@ -1,23 +1,22 @@
 version: '2'
 networks:
-  rra:
+  ram:
     external:
-      name: rra
+      name: ram
 services:
-  rra-postgis:
+  ram-postgis:
     image: mdillon/postgis:9.6
     ports:
       - "5432:5432"
     environment:
-      POSTGRES_PASSWORD: rra
-      POSTGRES_USER: rra
-      POSTGRES_DB: rra
+      POSTGRES_PASSWORD: ram
+      POSTGRES_USER: ram
+      POSTGRES_DB: ram
     volumes:
-      - ./setup/create-test-db.sh:/docker-entrypoint-initdb.d/create-test-db.sh
```

Comment:
Isn't the test db needed for when we run tests locally?

Reply:
Added this back.